pallets/nfts/src/lib.rs

```diff
@@ -1403,7 +1406,7 @@ pub mod pallet {
 		///
 		/// Origin must be the `ForceOrigin`.
 		///
-		/// - `owner`: The account granting approval for delegated transfer.
+		/// - `owner`: The owner of the approvals granted by the `origin`.
```

Comment:
This still doesn't make sense. The owner is not granted by the origin. We can change it back as I can not think of a better setence.